src/main.js

```diff
@@ -4,6 +4,8 @@ import 'focus-visible';
 // Internal Modules
 import initResponsiveVideo from './components/modular-content/responsive-video/responsive-video';
 import initToggle from './components/toggle/toggle';
+import initDownloadFilter from './components/download-list/download-list';
 
 initResponsiveVideo();
 initToggle();
+initDownloadFilter();
```

Comment:
Can't we make it a module that only loads on the `downloads` page?

Reply:
refactored to module